cmd/frontend/graphqlbackend/feature_flags.go

```diff
@@ -159,6 +159,20 @@ func (r *schemaResolver) OrganizationFeatureFlagValue(ctx context.Context, args
 	return result, nil
 }
 
+func (r *schemaResolver) OrganizationFeatureFlagOverrides(ctx context.Context) ([]*FeatureFlagOverrideResolver, error) {
+	user, err := backend.CurrentUser(ctx, r.db)
+	if err != nil {
+		return nil, err
+	}
+
+	flags, err := database.FeatureFlags(r.db).GetOrgOverridesForUser(ctx, user.ID)
```

Comment:
We are transitioning to a new and better way to use database, this is the new way:
```suggestion
	flags, err := r.db.FeatureFlags().GetOrgOverridesForUser(ctx, user.ID)
```